tests/test.js

```diff
@@ -1209,6 +1209,18 @@
                 same(get_props_without_distinct_id(mixpanel.test), props, "properties set properly");
             });
 
+            test('register_page_lifecycle', 3, function() {
```

Comment:
The second argument is `expected`, from the old version version of QUnit that this lib uses (and [removed in 2014](https://github.com/qunitjs/qunit/blob/master/History.md#1130--2014-01-04)). It specifies the number of assertions which are expected to be run during the test; for instance, this test will fail if only 2 test assertions get run.

As for clearing between pageloads, that's a pretty tough thing to test with QUnit. What I'd suggest is inspecting the contents of `mixpanel.test.persistence` - for instance, register one superproperty the normal way and another the `page_lifecycle` way, assert that the first is present in the persistent store and the second is not.